build/bakefiles/config.bkl

```diff
@@ -408,6 +408,7 @@ compiled .lib files and setup.h under the lib/ toplevel directory.
         <option name="WX_CFLAGS"/>
         <option name="WX_CXXFLAGS"/>
         <option name="WX_LDFLAGS"/>
+        <option name="WX_TEST_CXXFLAGS"/>
```

Comment:
I think it's a bit misleading to call it like this, it should probably be called `CATCH2_CXXFLAGS` or something like this instead.